### ISSUE TITLE
Allow Search option to Exclude Subscribed Topics

### ISSUE THREAD

**Issue by JPrevost** [MITLibraries/scoap3hub#77]
Use case: click saved search and only see matches that you are not subscribed to which should streamline the subscription process for new matched topics.

**Comment by richardrodgers** [comment]
Great idea - my question is whether this should be implemented as a filter on the search results returned from ES, or write it into the index itself. The latter would be more performant, but that really entangles the indexing to subscribers in a fairly complex way. I'd vote for filter first, then see if we need to do more. What do you think?

**Comment by JPrevost** [comment]
Yeah... it's a tough call. Doing it post search means really needing to get the full resultset instead of paginating or accepting the results pages may not always contain the requested per_page value (which is clearly bad UX).

I haven't really thought about how it might be done in elastic search...maybe something like:
"do this search but exclude these topic_ids" is doable... if so, that's maybe the way to go.

**Comment by JPrevost** [comment]
Regardless, I agree that trying to index the subscribers is probably too messy if we can avoid it.
